### PR TITLE
Update returns naming

### PR DESCRIPTION
**Pull request by roxy-dao**
returns naming hopefully I have them the right way around 🙏🏻

### CODE REVIEW

**Comment by CarlosNZ** [comment]
> hopefully I have them the right way around

Yes, if only there was a way for the name itself to indicate which way round they go!  Oh wait.... 👀

File: content/docs/distribution/customer_returns.md [hunk 1]
```diff
@@ -13,33 +13,33 @@ toc = true
 top = false
 +++
 
-An Inbound Returns is used to record stock which is returned by a customer.
+An Customer Returns is used to record stock which is returned by a customer.
```
Comment (lache-melvin) [inline content/docs/distribution/customer_returns.md:16]: Can we fix an -> a everywhere 🥺

**Review by lache-melvin — CHANGES_REQUESTED** [review]
Looks good! Just wanna fix all the `an`s hehe

File: content/docs/distribution/customer_returns.md [hunk 2]
```diff
@@ -13,33 +13,33 @@ toc = true
 top = false
 +++
 
-An Inbound Returns is used to record stock which is returned by a customer.
+a Customer Returns is used to record stock which is returned by a customer.
```
Comment (lache-melvin) [inline content/docs/distribution/customer_returns.md:16]: Hehe ok nit one more time... sentences should start with capital 🙈

File: content/docs/distribution/customer_returns.md [hunk 3]
```diff
@@ -13,33 +13,33 @@ toc = true
...
 
-It is important to be able to receive goods back from a customer without it being counted as goods coming from a supplier. If your customer issues the goods to you on an Outbound Shipment, and you then receive it through an Inbound Shipment, then those goods will be counted as part of your store's 'supply' from suppliers, which is not correct. An Inbound Return is the right way to receive goods back from a customer.
+It is important to be able to receive goods back from a customer without it being counted as goods coming from a supplier. If your customer issues the goods to you on an Outbound Shipment, and you then receive it through an Inbound Shipment, then those goods will be counted as part of your store's 'supply' from suppliers, which is not correct. a Customer Return is the right way to receive goods back from a customer.
```
Comment (lache-melvin) [inline content/docs/distribution/customer_returns.md:18]: here too

File: content/docs/replenishment/inbound_shipments.md [hunk 4]
```diff
@@ -526,7 +526,7 @@ Once you are happy with your quantities to return, you can click `Next step`. Th
 
 When you are happy with the quantities and reasons, you can press on:
 
-- the `OK` button. An Outbound Return will be created, and you will be redirected to view it. The quantities of stock selected to return will be reserved for the new return.
+- the `OK` button. a Supplier Return will be created, and you will be redirected to view it. The quantities of stock selected to return will be reserved for the new return.
```
Comment (lache-melvin) [inline content/docs/replenishment/inbound_shipments.md:529]: oop and here

File: content/docs/replenishment/supplier_returns.md [hunk 5]
```diff
@@ -13,31 +13,31 @@ toc = true
 top = false
 +++
 
-Outbound Returns can be used to return stock to a supplier.
+Supplier Returns can be used to return stock to a supplier.
 
-It is important to be able to return goods to a supplier without it being counted as goods being issued to a customer. If you just issue goods to your supplier through an Outbound Shipment, then those goods will be counted as part of your store's 'issuance' (consumption). An Outbound Return is the right way to return goods to a supplier.
+It is important to be able to return goods to a supplier without it being counted as goods being issued to a customer. If you just issue goods to your supplier through an Outbound Shipment, then those goods will be counted as part of your store's 'issuance' (consumption). a Supplier Return is the right way to return goods to a supplier.
```
Comment (lache-melvin) [inline content/docs/replenishment/supplier_returns.md:18]: one more 😁

**Review by lache-melvin — CHANGES_REQUESTED** [review]

**Review by lache-melvin — APPROVED** [review]
third time's the charm 🎉

**Pull request merged**